tutorials/getting_started/part3_deutsch_jozsa.ipynb

```diff
@@ -71,22 +70,14 @@
      "shell.execute_reply": "2024-05-07T14:27:01.549100Z"
     }
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Opening: https://platform.classiq.io/circuit/48398168-acd9-4aa9-a9f3-c148338fb4ee?version=0.41.0.dev39%2B79c8fd0855\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "from classiq import *\n",
     "\n",
     "\n",
     "@qfunc\n",
-    "def my_black_box_prediate(aux: QBit, target: QNum):\n",
-    "    aux ^= target > 30\n",
+    "def my_black_box_prediate(aux: QNum, target: QNum):\n",
```

Comment:
Why was this changed? I think a variable of type `QBit` actually makes more sense in storing the result of a boolean expression..